exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,19 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function upperCase(a) {
+    return a.toUpperCase();
+}
+let upperMentor1 = upperCase(mentor1)
+
+
+function greeting(a, b, c ,d ,e) {
+            
+    return "HELLO " +upperMentor1+"\n" + "HELLO " +b+ "\n"+ "HELLO " +c+"\n"+"HELLO " +d+"\n" +"HELLO " +e+"\n";
```

Comment:
Good attempt but this isn't quite a nested function. The upperCase function should be called in the greeting function. This would make a upperCase a nested function as it is called inside another function.